src/main/front/components/Modal/Signup/index.js

```diff
@@ -0,0 +1,2 @@
+import Signup from './Signup.js';
```

Comment:
We don't need the index.js files. When we need to import the function, we can just do it from the component level